gm4/plugins/readme_generator.py

```diff
@@ -0,0 +1,189 @@
+from beet import Context, TextFile
+import re
+from pathlib import Path
+
+global_replacements = {
+    r"https:\/\/wiki\.gm4\.co\/wiki\/([\w_]+)": r"https:MY_URL\\\1",
+    }
+
+def beet_default(ctx: Context):
+    """Loads the README.md and modifies:
+        - converts local images to URLs pointed at the repo
+        - download links for respective download sites
+        - pulls credits from beet.yaml and contributors.json
+        - pulls YT link from beet.yaml
+        - performs some site-specific replacements
+        - generates a BBCode translation of the markdown file"""
+    readme_path = Path(ctx.project_id) / "README.md"
+    if not readme_path.exists():
+        return
+    global_readme = TextFile(source_path=readme_path)
+    global_contents = global_readme.text
+    running_readme_gen = ctx.meta.get("readme-gen", False)
```

Comment:
Why is this flag necessary?

Reply:
The PMC post generation has alot of replacement rules. Since there is no way to automate uploading the README to PMC right now, and the PMC readme is otherwise not saved anywhere on a normal beet-release build, this flag just disables that file generation. Only when running the custom beet command will PMC readmes get produced.

Reply:
Then I would rename it to mention `pmc` or add some comments